### PR TITLE
Documentation: some rewording and nitpicks

### PR DESCRIPTION
**Pull request by aalemayhu**
These are just suggestions that sound better to me. Let me know if I should reword more or drop any. Please see individual patches for more detail.

Thanks.

### CODE REVIEW

File: Documentation/intro.rst [hunk 1]
```diff
@@ -24,23 +24,22 @@ service-oriented architecture often referred to as *microservices*, wherein a
...
-
-Traditional Linux network security approaches (e.g., iptables) filter on IP
-addresses and TCP/UDP ports. The highly volatile lifecyle of containers causes
-these approaches to struggle to scale side by side with the application as load
-balancing tables and access control lists carrying hundreds of thousands of
-rules that  need to be update with a continuously growing frequency. Protocol
-ports (e.g. TCP port 80 for HTTP traffic) can no longer be used to
-differentiate between application traffic for security purposes as the port is
-utilized for a wide range of messages across services
+microservices environments and traditional Linux network security approaches
```
Comment (aanm) [inline Documentation/intro.rst]: `traditional Linux...` is repeated

File: Documentation/gettingstarted.rst [hunk 2]
```diff
@@ -51,25 +51,19 @@ repository:
 Step 2: Starting the Docker + Cilium VM
 ---------------------------------------
 
-Open a terminal and navigate into the top of the cilium source directory.
```
Comment (aanm) [inline Documentation/gettingstarted.rst]: The visual of a console font helps me to see what are the commands I need to run without reading the whole paragraph. I like better the way it look.

File: Documentation/gettingstarted.rst [hunk 3]
```diff
@@ -51,25 +51,19 @@ repository:
...
 connection. Vagrant will set up a VM, install the Docker container runtime and
 run Cilium with the help of Docker compose. When the script completes successfully,
 it will print:
 
 ::
+
     ==> cilium-1: Creating cilium-kvstore
     ==> cilium-1: Creating cilium
     ==> cilium-1: Creating cilium-docker-plugin
-    $
```
Comment (aanm) [inline Documentation/gettingstarted.rst]: The `$` signals the user won't see anymore messages besides those 3.

**Review by aanm — CHANGES_REQUESTED** [review]

File: Documentation/intro.rst (hunk 1 above)
Comment (aalemayhu) [reply Documentation/intro.rst]: Should be fixed but please verify.

File: Documentation/gettingstarted.rst (hunk 2 above)
Comment (aalemayhu) [reply Documentation/gettingstarted.rst]: Reverted changes.

File: Documentation/gettingstarted.rst (hunk 3 above)
Comment (aalemayhu) [reply Documentation/gettingstarted.rst]: ah, reverted.

File: Documentation/intro.rst [hunk 4]
```diff
@@ -24,20 +24,17 @@ service-oriented architecture often referred to as *microservices*, wherein a
...
-microservices environments and
-
-Traditional Linux network security approaches (e.g., iptables) filter on IP
-addresses and TCP/UDP ports. The highly volatile lifecyle of containers causes
+address and TCP/UDP ports, but IP addresses frequently churn in dynamic
+microservices environments. The highly volatile life cycle of containers causes
 these approaches to struggle to scale side by side with the application as load
 balancing tables and access control lists carrying hundreds of thousands of
-rules that  need to be update with a continuously growing frequency. Protocol
+rules that  need to be updated with a continuously growing frequency. Protocol
```
Comment (aanm) [inline Documentation/intro.rst]: `that  need`
2 spaces between `that` and `need`

**Review by aanm — DISMISSED** [review]

File: Documentation/intro.rst (hunk 4 above)
Comment (aalemayhu) [reply Documentation/intro.rst]: Fixed.

**Review by tgraf — APPROVED** [review]

**Pull request merged**